src/lib/schematics/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.__styleext__

```diff
@@ -4,5 +4,9 @@
 
 .sidenav {
   width: 200px;
-  box-shadow: 3px 0 6px rgba(0,0,0,.24);
+}
+
+.mat-toolbar.mat-primary {
+  position: sticky;
```

Comment:
Should we really be using `position: sticky`? The browser support isn't great and `fixed` should be able to get the job done for an example.

Reply:
We actually use position sticky on the material docs site. Supports seems decent just missing IE11.